app/Http/Integrations/GithubConnector.php

```diff
@@ -23,4 +31,27 @@ protected function defaultHeaders(): array
             "Accept" => "application/json",
         ];
     }
+
+    protected function resolveLimits(): array
+    {
+        return [
+            Limit::allow(config("services.rate_limit"))->everyHour(),
+        ];
+    }
+
+    protected function resolveRateLimitStore(): RateLimitStore
+    {
+        return new LaravelCacheStore(Cache::store());
+    }
+
+    protected function handleTooManyAttempts(Response $response, Limit $limit): void
+    {
+        if ($response->status() !== 429 && $response->status() !== 403) {
+            return;
+        }
```

Comment:
```suggestion
        if (in_array($response->status(), [Status::HTTP_TOO_MANY_REQUESTS, Status::HTTP_FORBIDDEN]) {
            return;
        }
```

Lets use `Symfony\Component\HttpFoundation\Response` codes.